src/lib/Unpack.ts

```diff
@@ -14,6 +14,7 @@ export class Unpack {
         s: 1,
         f: 4,
         d: 8,
+        T: 8,
```

Comment:
The `Unpack` class is based around python's **struct** lib, so the valid type here should be `Q` instead of `T`
```suggestion
        Q: 8,
```